Added Kubernetes service resolver to loadbalancing exporters. (#5678)
grafana/agent: docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

`timeout`  | `duration` | Resolver timeout. | `"1s"`  | no
`port`     | `string`   | Port to be used with the IP addresses resolved from the DNS hostname. | `"4317"` | no

### k8s block

> The `k8s` block provides a way to load balance across the pods of a Kubernetes service. The Agent will be notified

Reviewer: ```suggestion
You can use the `k8s` block to load balance across the pods of a Kubernetes service. The Agent will be notified
```
